docs/ext/pydantic_autosummary/generate.py

```diff
@@ -0,0 +1,880 @@
+        self.env.filters["underline"] = _underline
+
+        if app.translator:
+            self.env.add_extension("jinja2.ext.i18n")
+            # ``install_gettext_translations`` is injected by the
+            # ``jinja2.ext.i18n`` extension
+            self.env.install_gettext_translations(app.translator)  # type: ignore[attr-defined]
+
+    def render(self, template_name: str, context: dict) -> str:
+        """Render a template file."""
+        try:
+            template = self.env.get_template(template_name)
+        except TemplateNotFound:
+            try:
+                # objtype is given as template_name
+                template = self.env.get_template("autosummary/%s.rst" % template_name)
+            except TemplateNotFound:
+                # fallback to base.rst
+                template = self.env.get_template("autosummary/base.rst")
+
+        return template.render(context)
+
+
+def _split_full_qualified_name(name: str) -> tuple[str | None, str]:
+    """Split full qualified name to a pair of modname and qualname.
+
+    A qualname is an abbreviation for "Qualified name" introduced at PEP-3155
+    (https://peps.python.org/pep-3155/).  It is a dotted path name
+    from the module top-level.
+
+    A "full" qualified name means a string containing both module name and
+    qualified name.
+
+    .. note:: This function actually imports the module to check its existence.
+              Therefore you need to mock 3rd party modules if needed before
+              calling this function.
+    """
+    parts = name.split(".")
+    for i, _part in enumerate(parts, 1):
+        try:
+            modname = ".".join(parts[:i])
+            importlib.import_module(modname)
+        except ImportError:
+            if parts[: i - 1]:
+                return ".".join(parts[: i - 1]), ".".join(parts[i - 1 :])
+            return None, ".".join(parts)
+        except IndexError:
+            pass
+
+    return name, ""
+
+
+# -- Generating output ---------------------------------------------------------
+
+
+class ModuleScanner:
+    def __init__(self, app: Any, obj: Any) -> None:
+        self.app = app
+        self.object = obj
+
+    def get_object_type(self, name: str, value: Any) -> str:
+        return get_documenter(self.app, value, self.object).objtype
+
+    def is_skipped(self, name: str, value: Any, objtype: str) -> bool:
+        try:
+            return self.app.emit_firstresult(
+                "autodoc-skip-member", objtype, name, value, False, {}
+            )
+        except Exception as exc:
+            # ----------- pydantic_autosummary change
+            logger.warning(
+                __(
+                    "pydantic_autosummary: failed to determine %r to be documented, "
+                    "the following exception was raised:\n%s"
+                ),
+                name,
+                exc,
+                type="autosummary",
+            )
+            # ----------/ pydantic_autosummary change
```

Comment:
Just changed the logger to use `pydantic_autosummary: ` instead of `autosummary: `